rct229/schema/ASHRAE229.schema.json

```diff
@@ -1940,10 +1940,25 @@
                 },
                 "cooling_design_day_type": {
```

Comment:
Lines 1941-1950 must be deleted when the schema is updated. I didn't do this in this PR because deleting the objects requires to change other rules and files as well.